cpg_workflows/query_modules/seqr_loader.py

```diff
@@ -48,7 +48,7 @@ def annotate_cohort(
         force_bgz=True,
         array_elements_required=False,
     )
-    mt = checkpoint_hail(mt, 'mt-imported-vcf.mt', checkpoint_prefix)
+    mt.checkpoint(output=str(checkpoint_prefix) + 'mt-imported.mt', overwrite=True)
```

Comment:
```suggestion
    mt.checkpoint(output=str(checkpoint_prefix / 'mt-imported.mt'), overwrite=True)
```

this will work but the current checkpoint path will be a bit dodgy looking

Reply:
Unfortunately linter doesn't like this 
```
mypy.....................................................................Failed
- hook id: mypy
- exit code: 1

cpg_workflows/scripts/mt_to_es_without_dataproc.py:259: note: By default the bodies of untyped functions are not checked, consider using --check-untyped-defs  [annotation-unchecked]
cpg_workflows/query_modules/seqr_loader.py:51: error: Unsupported left operand
type for / ("str")  [operator]
        mt.checkpoint(output=str(checkpoint_prefix / 'mt-imported.mt'), ov...
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cpg_workflows/query_modules/seqr_loader.py:51: error: Unsupported left operand
type for / ("None")  [operator]
        mt.checkpoint(output=str(checkpoint_prefix / 'mt-imported.mt'), ov...
                                 ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
cpg_workflows/query_modules/seqr_loader.py:51: note: Left operand is of type "str | None"
```